src/data/script.tsx

```diff
@@ -146,7 +146,7 @@ const conversation: IMessage[] = [
   {
```

Comment:
Too high up in this diff but on L75 with the MHRA acronyms, might be able to get some bonus funny points if we can come up with something rust related 🤔 Will have a think haha

Reply:
Nice Idea!